dockers/docker-snmp-sv2/Dockerfile.j2

```diff
@@ -57,15 +59,15 @@ RUN python3.6 -m pip install --no-cache-dir \
 {{ copy_files("python-wheels/", docker_snmp_sv2_whls.split(' '), "/python-wheels/") }}
 
 # Install locally-built Python wheel dependencies
-{{ install_python_wheels(docker_snmp_sv2_whls.split(' ')) }}
+{{ install_python3_wheels(docker_snmp_sv2_whls.split(' ')) }}
 {% endif %}
 
-RUN python3.6 -m sonic_ax_impl install
+RUN python3 -m sonic_ax_impl install
 
 # Clean up
 RUN apt-get -y purge     \
-        libpython3.6-dev \
-        libpython3.6     \
+        libpython3.7-dev \
+        libpython3.7     \
```

Comment:
You did not install them, Actually you need `purge python3.7-dev`

Reply:
libpython3.7 is getting automatically installed. Since libpython3.6 was being removed previously, I added this change.

For now retaining libpython3.7.

Reply:
The original code is not perfect, the right package to purge is `python3-dev`, which explicitly installed above. Could you help test?

---
In reply to: [419825978](https://github.com/Azure/sonic-buildimage/pull/4529#discussion_r419825978) [](ancestors = 419825978)

Reply:
Addressed.